src/experiments/class-collect-cta.js

```diff
@@ -0,0 +1,10 @@
+export default {
+  name: 'class-collect-cta',
+  experimentID: 'cljcOGr3RX-jUiSHCaMu-A',
+  isEligible: ({ route }) => {
+    const parts = route.name.split('___');
+    const routeName = parts[0];
```

Comment:
I think it is more robust to use `getRouteBaseName()` from context to extract the route name without localization

You may need to inject `getRouteBaseName` from [here](https://github.com/likecoin/liker-land/blob/57c8784c02c880150c75ae10f7c541cb4aa2c6b7/src/mixins/experiment.js#L14) in order to use it